service PredictorGrpc {
    rpc PlayerPrediction(PlayerInput) returns (PredictionOutput) {}
    rpc TeamPrediction(TeamInput) returns (PredictionOutput) {}
    rpc ChampionshipPrediction(ChampionshipInput) returns (PredictionOutput) {}
}

message PlayerInput {
    required int32 firstPlayerId = 1;
    required int32 secondPlayerId = 2;
}

message TeamInput {
    required int32 firstTeamId = 1;
    required int32 secondTeamId = 2;
}

message ChampionshipInput {
    required int32 id = 1;
}

message PredictionOutput {
    required int32 id = 1;
}